### ISSUE TITLE
[Bug]: Call RWKV6Attention and report an error  environment.

### ISSUE THREAD

**Issue by synbol** [sustcsonglin/flash-linear-attention#74]
### Describe the bug

**Error message:**
python: /project/lib/Analysis/Allocation.cpp:47: std::pair<llvm::SmallVector<unsigned int>, llvm::SmallVector<unsigned int> > mlir::triton::getCvtOrder(mlir::Attribute, mlir::Attribute): Assertion `!(srcMmaLayout && dstMmaLayout && !srcMmaLayout.isAmpere()) && "mma -> mma layout conversion is only supported on Ampere"' failed.

### Steps to reproduce the bug

**Calling process:**
from fla.layers.rwkv6 import RWKV6Attention
self.attention = RWKV6Attention(hidden_size=config.dim, num_heads=config.n_head)

o, _, past_key_values = self.attention(self.attention_norm(x), attention_mask=mask, past_key_values=past_key_values)

### Expected behavior

None

### Environment info

**Environment:**
torch    2.4.1
triton    3.0.0
einops  0.8.0

**Comment by synbol** [comment]
self.attention = RWKV6Attention(hidden_size=config.dim, num_heads=config.n_head, layer_idx=layer_id)

**Comment by yzhangcs** [comment]
Hi, can you provide some runnable code for reproduction. It works normally for me by running [this](https://github.com/sustcsonglin/flash-linear-attention/blob/main/benchmarks/benchmark_training_throughput.py)
```sh
python benchmark_training_throughput.py --name rwkv6
```

**Comment by sustcsonglin** [comment]
Hi, thanks for reporting it. What is your GPU model?
